lib/profile/sections/sections.dart

```diff
@@ -18,10 +16,11 @@ class ProfileSections extends StatelessWidget {
 
     return 
     Container(
-      height: MediaQuery.of(context).size.height - profileHeaderHeight - appNavBarHeight,
-      child: SingleChildScrollView(
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
+      child: Flexible(
+        child: ListView(
+          padding: EdgeInsets.all(0.0),
```

Comment:
What does the padding do? I commented it out and didn't notice a difference visually.